packages/ui5-application-writer/src/index.ts

```diff
@@ -39,7 +39,18 @@ async function generate(basePath: string, ui5AppConfig: Ui5App, fs?: Editor): Pr
         ignore.push('**/gitignore.tmpl');
     }
 
-    fs.copyTpl(join(tmplPath, 'core', '**/*.*'), join(basePath), ui5App, undefined, {
+    // Determine the framework URL and version based on the project type, if project is not EDMX  include ui5App.ui5.frameworkUrl & ui5App.ui5.version
+    const frameworkUrl = !isEdmxProjectType && ui5App.ui5.frameworkUrl ? ui5App.ui5.frameworkUrl + '/' : undefined;
+    const version = !isEdmxProjectType && ui5App.ui5.version ? ui5App.ui5.version + '/' : undefined;
```

Comment:
I think you should ignore the version if there is no framework url because something like `/1.172.1/resources/...` wouldn't work anywhere.